### PR TITLE
Add docs for default schema names and delta updates

### PR DESCRIPTION
**Pull request by danthelion**
Docs for feature that was recently shipped. Discussion here: https://estuaryworkspace.slack.com/archives/C02G25WJCNS/p1733421953163009

- - -
This change is [Reviewable](https://reviewable.io/reviews/estuary/flow/1812)

### CODE REVIEW

File: site/docs/concepts/materialization.md [hunk 1]
```diff
@@ -317,6 +317,17 @@ A given endpoint may support standard updates, delta updates, or both.
 This depends on the [materialization connector](../reference/Connectors/materialization-connectors/README.md). Expect that a connector will use
 standard updates only unless otherwise noted in its documentation.
 
+### Delta updates for every binding in a Materialization
+
+When configuring a materialization, you can set delta updates as the materialziation mechanism
```
Comment (aeluce) [inline site/docs/concepts/materialization.md:322]: Typo on "materialization" here (materialziation).

File: site/docs/concepts/materialization.md [hunk 2]
```diff
@@ -317,6 +317,17 @@ A given endpoint may support standard updates, delta updates, or both.
...
+for every binding. Afterwards, you are able to modify this setting individually for each binding
+for further customization.
+
+:::note
+This functionality is not supported for all Materialization connectors. Consult the individual connector
+documentation for details.
```
Comment (aeluce) [inline site/docs/concepts/materialization.md:328]: Can we be more specific in this note and in the schema name note on what to look for? Something like: "This functionality is only supported for materialization connectors with [xyz]. Consult the individual connector documentation for details."

File: site/docs/concepts/materialization.md (hunk 1 above)
Comment (danthelion) [reply site/docs/concepts/materialization.md]: good catch, thanks!

File: site/docs/concepts/materialization.md (hunk 2 above)
Comment (danthelion) [reply site/docs/concepts/materialization.md]: Clarified a bit, let me know if it makes sense

**Review by aeluce — APPROVED** [review]
Thanks for updating; LGTM!

**Pull request merged**